hammurabi/mixins.py

```diff
@@ -202,17 +249,14 @@ def generate_pull_request_body(self, pillar) -> str:
         ]
 
         for law in pillar.laws:
+            if not [rule for rule in law.rules if rule.made_changes]:
```

Comment:
in general I dislike negative conditions and `continue`, so I suggest:
```
if [rule for rule in law.rules if rule.made_changes]:
    body.append(f"\n### {law.name}")
    .....
```

Reply:
Simplified the logic. Only those laws will run which has rules which made modifications or failed